assets/javascripts/discourse/initializers/random-hello-bar.js.es6

```diff
@@ -0,0 +1,160 @@
+export default {
+  name: 'random-hello-bar',
+  initialize() {
+
+
+    (function(window, $) {
+      "use strict";
+
+      var EXPIRE_DAYS = 14;
+
+      var RandomHelloBar = function() {
+
+        var isEnabled = true;
+
+        // if jQuery Cookie is available, we may have disabled ourselves for the number of days set in EXPIRE_DAYS if user manually closed the bar
+        if ($.cookie) {
+          if ($.cookie('awesomeBarDisabled')) {
```

Comment:
yeah we could. The whole thing will be refactored shortly to remove jQuery all together